shared/macros-oval.jinja

```diff
@@ -0,0 +1,64 @@
+{{%- macro oval_lineinfile(msg='', path='', parameter='', value='', default='', application='', rpm_package='') -%}}
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    <metadata>
+      <title>{{{ msg or rule_title }}}</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
+      <description>{{{ rule_title }}}</description>
+    </metadata>
+    <criteria comment="{{{ application }}} is configured correctly or is not installed"
+    operator="OR">
+      <criteria comment="{{{ application }}} is not installed" operator="AND">
+        <extend_definition comment="{{{ application }}} is not required or requirement is unset"
+        definition_ref="{{{ application }}}_not_required_or_unset" />
+        <extend_definition comment="rpm package {{{ rpm_package }}} removed"
+        definition_ref="package_{{{ rpm_package }}}_removed" />
+      </criteria>
+      <criteria comment="{{{ application }}} is installed and configured" operator="AND">
+        <extend_definition comment="{{{ application }}} is required or requirement is unset"
+        definition_ref="{{{ application }}}_required_or_unset" />
+        <extend_definition comment="rpm package {{{ rpm_package }}} installed"
+        definition_ref="package_{{{ rpm_package }}}_installed" />
+        {{%- if default %}}
+        <criteria comment="{{{ application }}} is configured or has default value" operator="OR">
+        {{%- endif %}}
+          <criterion comment="Check {{{ parameter }}} in {{{ path }}}"
+          test_ref="test_{{{ rule_id }}}" />
+        {{%- if default %}}
+          <criterion comment="Check for default value of {{{ parameter }}} in {{{ path }}}"
```

Comment:
The comment is misleading to me, the test is checking that the config file is not overriding the default value.
Not that the default value is set in the config file.

Reply:
Updated to `Check absence of parameter`.